libretro/libretro.c

```diff
@@ -591,6 +591,7 @@ void update_controllers()
 static void update_variables(bool startup)
 {
     struct retro_variable var;
+    const char *screen_size_key = NULL;
```

Comment:
I think this might be unitialized on a 2nd not `startup` call (happens when core options are changed.. after start, but most wont re-apply), so `var.key` in line +1048 will request with `null`, but since we don't update it, it causes no side-effect.
Could you confirm this behaviour? We probably want to make it `static`, I dont like asking for some NULL key if thats the case.

Reply:
I don't know how to trigger the 2nd `startup` call. I tried messing with the core options when running, a lot of times, but weren't able to trigger this function again. Also tried stopping the core and opening again (with and w/o option changed) with no success. Also I have been using this branch a lot, and I haven't had any issues.

But, I'm with you about not requesting `NULL` keys, that can't go well. 

I'll make that variable `static`, and also assign to it "-43screensize" instead of `NULL`, so it will always request a valid key.

Reply:
I'll mark this as resolved.